Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCRtpSenderBackend.cpp

```diff
@@ -123,6 +123,55 @@ RTCRtpSendParameters LibWebRTCRtpSenderBackend::getParameters() const
     return toRTCRtpSendParameters(*m_currentParameters);
 }
 
+static bool validateModifiedParameters(const RTCRtpSendParameters& newParameters, const RTCRtpSendParameters& oldParameters)
```

Comment:
It seems like this code would be better placed in RTCRtpSendParameters::operator==

Reply:
RTCRtpSendParameters::operator== would need to check all fields (read-only and read/write).
In our case, we are only comparing read-only properties.